upup/pkg/kutil/import_cluster.go

```diff
@@ -507,18 +506,18 @@ func (x *ImportCluster) ImportAWSCluster() error {
 	return nil
 }
 
-func parseInt(s string) (int, error) {
```

Comment:
Some commented files refer to this function. So comment this function not remove it